### DISCUSSION TITLE
Stereo with max98357a

### DISCUSSION THREAD

**Discussion by MisterAG** [schreibfaul1/ESP32-audioI2S#421] [Q&A] [answered]
Hi,

could someone briefly explain how I can use two max98357a, to create a stereo webradio?

I just wire the pins of the two boards, to the same pins on the ESP32. And initialise audio as usual in code?
(I haven't ordered my boards yet, so I can't test. I want to be sure what I want is possible)

Best regards,
Jan

**Comment by schreibfaul1** [comment]
Yes, that is possible. This is a mono amplifier, two are required for stereo. By default, I2S transmits the stereo signal. To set the MAX98357a to a channel, depending on the manufacturer, a solder bridge is placed or an additional resistor is soldered.

**Comment by MisterAG** [comment, marked as answer]
I think I found the solution in my case after some extensive googeling, by setting the correct voltage on the SD pin:

This pin is used for shutdown mode but is also used for setting which channel is output. It's a little confusing but essentially:

If SD is connected to ground directly (voltage is under 0.16V) then the amp is shut down
If the voltage on SD is between 0.16V and 0.77V then the output is (Left + Right)/2, that is the stereo average. 
If the voltage on SD is between 0.77V and 1.4V then the output is just the Right channel
If the voltage on SD is higher than 1.4V then the output is the Left channel.
